main.cpp

```diff
@@ -270,8 +550,36 @@ Thing 10) Synthesizer
  send me a DM to review your pull request when the project is ready for review.
 
  Wait for my code review.
+
  */
 
+ struct Synthesizer       
+{
+    //has oscillator
+    Oscillator oscillator; //3) member variables with relevant data types.
+    //has adsr
+    Adsr adsr; //3) member variables with relevant data types.
+    //has filter
+    Filter filter; //3) member variables with relevant data types.
+    //has midi
+    Midi midi; //3) member variables with relevant data types.
+    //has lfo
+    Lfo lfo; //3) member variables with relevant data types.
+
+    /* 
+    member functions with a user-defined type as the parameter.
+    The user-defined type parameter happens to be the nested class.
+    */
+
+    //2) create sounds
+    void createSounds();
+    //2) send midi
+    void sendMidi();
+    //2) manipulate the sound with adsr evelopes
+    void manipulateSoundAdsr();
```

Comment:
- [ ] add params that are related to what these functions are supposed to do based on their description.